se_apps/CMakeLists.txt

```diff
@@ -151,3 +151,12 @@ foreach(appname ${BUILT_LIBS})
   add_version(${appname}
       "${common_libraries}")
 endforeach()
+
+
+
+# ------- INSTALL -----------------------
+INSTALL(
+        DIRECTORY include/
+        DESTINATION /usr/local/include/se
+        FILES_MATCHING PATTERN "*.h*"
+)
```

Comment:
I don't think the headers from se_apps are needed for programs using supereight. Did you need them in your case?

Reply:
yes the default_parameters.h is used and the thirdparty/vector_types.h

Reply:
Hi Anna, first of all thanks a lot for the PR, great job! Can you point me to where the default_parameters and vector_types.h are used in se_core and se_denseslam?

Reply:
Thanks, I am using these files in my ROS wrapper for the supereight [WIP]

Reply:
The problem with the se_apps headers is that they shouldn't be redistributed. I can give you more details on that offline, but for the moment it would be better if we can exclude them from the install. Any dependance on vector_types.h is deprecated and we should aim at removing that header completely.